hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -2567,6 +2573,17 @@ public ReportProcedureDoneResponse reportProcedureDone(RpcController controller,
     return ReportProcedureDoneResponse.getDefaultInstance();
   }
 
+  private void throwOnOldMasterStartCode(Stream<Long> masterStartCodeStream)
```

Comment:
Could use LongStream here, but for me I prefer we log more about the procedures here, like which procedure's master start code is greater than current master's start code, and also log the start codes for better debugging.